silk/node/fifteen_four_dev_board.py

```diff
@@ -169,6 +170,9 @@ def set_logger(self, parent_logger):
     def log_debug(self, log_line):
         if self.logger is not None:
             self.logger.debug(log_line)
+        
+        if self.otns_manager is not None:
+            self.otns_manager.check_for_extaddr_update(self, log_line)
```

Comment:
What would be a better option? I also feel that this might not be the right place, but the issue is that the `getprop` response isn't logged by the `wpantund` process, but by the node itself.

Reply:
Should OTNS be monitoring all syslogs?

Reply:
Adding request to get extaddr